pitch_bad_alpha_all.py

```diff
@@ -0,0 +1,43 @@
+import argparse
+from pathlib import Path
+from astropy.table import Table, vstack
+from cxotime import CxoTime
+import check_fss
+
+
+def get_options():
+    parser = argparse.ArgumentParser(description="Make long plot")
+    parser.add_argument("--out",
+                        default=".",
```

Comment:
For HEAD processing, I suppose I could output to /proj/sot/ska/data/fss_check3/fss_prim ?  Though I suppose there is already a bad alpha text file in there.